src/lib/entities/extractors/cloud_function.py

```diff
@@ -86,6 +86,6 @@ def _cloud_function_resp_to_monitored_entities(page: Dict[Text, Any], svc_def: G
 @entity_extractor("cloud_function", "cloudfunctions.googleapis.com")
 async def get_cloud_function_entity(ctx: MetricsContext, project_id: str, svc_def: GCPService) -> Iterable[Entity]:
     """ Retrieve entity info on GCP cloud functions from google api. """
-    url = f"https://cloudfunctions.googleapis.com/v1/projects/{project_id}/locations/-/functions"
+    url = f"https://cloudfunctions.googleapis.com/v2/projects/{project_id}/locations/-/functions"
```

Comment:
What about V1 functions after this change?

Reply:
Hi, As I checked v2 endpoint returns both v1 and v2 functions